### PR TITLE
Session and visits page styling

### PR DESCRIPTION
**Pull request by DuarteOliveira8**
Links for preview:
- [Networking session](https://deploy-preview-44--friendly-shockley-474156.netlify.com/sessions/sessao-de-networking)
- [Cocktail session](https://deploy-preview-44--friendly-shockley-474156.netlify.com/sessions/pitch-de-oportunidades)
- [Visits](https://deploy-preview-44--friendly-shockley-474156.netlify.com/visits/example/)

### CODE REVIEW

**Comment by smbea** [comment]
The button to the tickets should say it in Portuguese as the rest of the website content. Also the square containing the logo in the visits template should always be an actual square

File: src/templates/session.js [hunk 1]
```diff
@@ -1,46 +1,35 @@
...
-
-  const {
-    title,
-    day,
-    start_time,
-    end_time,
-    place,
-    registration,
-  } = session.frontmatter
+  console.log(session.frontmatter)
```
Comment (pedromsfernandes) [inline src/templates/session.js]: This shouldn't be here!

File: src/components/session/companies.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import Img from "gatsby-image"
+import { graphql } from "gatsby"
+import { Container, Row, Col } from "reactstrap"
+
+import CompaniesStyles from "../../styles/session/companies.module.css"
+import { useStaticQuery } from "gatsby"
+
+// const getCompaniesArrayString = companies => {
```
Comment (pedromsfernandes) [inline src/components/session/companies.js]: Is this code needed for later?

File: src/templates/session.js (hunk 1 above)
Comment (DuarteOliveira8) [reply src/templates/session.js]: Sorry it was for testing

File: src/components/session/companies.js (hunk 2 above)
Comment (DuarteOliveira8) [reply src/components/session/companies.js]: No, I don't need it anymore. It's now deleted! Thanks!

**Pull request merged**